notifications_utils/recipient_validation/postal_address.py

```diff
@@ -146,6 +146,19 @@ def has_invalid_characters(self):
             line.startswith(tuple(self.INVALID_CHARACTERS_AT_START_OF_ADDRESS_LINE)) for line in self.normalised_lines
         )
 
+    @property
+    def has_no_fixed_abode_address(self):
+        """
+        We don't want users to sent to no fixed abode addresses, so validate that
+        - no lines just consist of "NFA" or "NFA," (case insensitive)
+        - the address does not contain "no fixed abode" or "no fixed address" (case insensitive)
+        """
+        if any(re.search(r"^nfa,?$", line, re.IGNORECASE) for line in self.normalised_lines):
```

Comment:
Normalised lines will never have trailing commas:
https://github.com/alphagov/notifications-utils/blob/0a04a4f126bb172245195eee9fe120d6ae87469f/notifications_utils/recipient_validation/postal_address.py#L37

Reply:
Good spot, I've simplified that line now since it's doesn't need to check for a trailing comma